clients/python/client/osparc/_studies_api.py

```diff
@@ -0,0 +1,74 @@
+from osparc_client import StudiesApi as _StudiesApi
+
+from ._utils import dev_feature
+
+
+class StudiesApi(_StudiesApi):
+    """Class for interacting with solvers"""
+
+    @dev_feature
+    def clone_study(self, study_id, **kwargs):
```

Comment:
For me this is ok for now, but wouldn't it be better to automate this?
(Or one could get all the methods from super, or if one wants to limit it, just provide a list with names of methods, and not writing all the procedural code?)
If this automation already exists, and this file is a result of it, I would not commit the resulting file in the repo.

Reply:
Good point. I was too lazy to do it, but I have fixed that now :smile

Reply:
Ok, great. Lazy is good, but I'm mostly too lazy to do you first approach  😄